src/depackers/stb_vorbis.c

```diff
@@ -1243,23 +1606,14 @@ static uint32 get_bits(vorb *f, int n)
          f->valid_bits += 8;
       }
    }
-   if (f->valid_bits < 0) return 0;
+
+   assert(f->valid_bits >= n);
```

Comment:
🙄

Reply:
This appears to have been a deliberate change upstream: https://github.com/nothings/stb/pull/935

Reply:
Got it—I didn't check the surrounding code, the check above that is the one that actually matters.